main/resource-access.bs

```diff
@@ -67,13 +69,35 @@ if it needs to access resources requiring authentication (see [[#webid]]).
 When it receives a response with a `403` or `404` status code,
 it MAY repeat the request with different credentials.
 
+A Solid client MUST use the `Content-Type` request header in `PUT`, `POST` and `PATCH` requests. [[Source](https://github.com/solid/specification/issues/70#issuecomment-547924171)]
 
-## Linked Data Platform ## {#ldp}
+## Uniform Resource Identifier ## {#uri}
 
-Issue: Write Linked Data Platform section.
+### Required shared slash semantics ### {#uri-path-component}
 
-Draft:
-A Solid data pod MUST conform to the LDP specification [[!LDP]].
+Server and client share the slash semantics in the URI path component representing hierarchy [[!RFC3986]]. [[Source](https://github.com/solid/specification/issues/35#issuecomment-547949014)]
```

Comment:
…, that is: (explain exactly what this entails and perhaps also does not entail).

Reply:
Revised: https://github.com/solid/specification/pull/157/commits/2abbeca6554a22529aae45b4eb925bf6729b8d3b . Entailment is further clarified in the other sections.